src/main/java/com/sparta/popupstore/domain/point/service/PointService.java

```diff
@@ -11,4 +25,45 @@ public class PointService {
 
     private final PointChargedLogRepository pointChargedLogRepository;
     private final PointUsedLogRepository pointUsedLogRepository;
+    private final PopupStoreRepository popupStoreRepository;
+
+    public PointChargeResponseDto pointCharge(User user, PointChargeRequestDto chargeRequest) {
+      if (chargeRequest.getChargedPoint() <= 1000) {
+        throw new CustomApiException(ErrorCode.LACK_OF_POINT);
+      }
+
+      PointChargedLog chargedLog = chargeRequest.toEntity(user);
+      user.ChargePoint(chargeRequest.getChargedPoint());
+      chargedLog = pointChargedLogRepository.save(chargedLog);
```

Comment:
```suggestion
      PointChargedLog chargedLog = chargeRequest.toEntity(user);
      chargedLog = pointChargedLogRepository.save(chargedLog);
      user.ChargePoint(chargeRequest.getChargedPoint());
```
잘못하면 로직 순서가 꼬여서 문제가 생길 수 있습니다.
굳이 필요한 이유가 없다면 이러한 순서가 좋을 것 같습니다.